Sources/tuist/TuistApp.swift

```diff
@@ -9,10 +9,22 @@ import TuistSupport
 @_documentation(visibility: private)
 private enum TuistServer {
```

Comment:
I think this can be called `TuistApp`, I don't know why it was called `TuistServer`, but it's definitely not accurate anymore. Sorry for the fly-by change request, I know it's not directly related to your PR!